src/Lykke.Common.ApiLibrary/Swagger/FormFileUploadOperationFilter.cs

```diff
@@ -49,6 +55,44 @@ public void Apply(Operation operation, OperationFilterContext context)
 
                 operation.Parameters[parameterIndex] = newParameter;
             }
+
+            var notGreedyDescriptions = descriptions
+                .Where(x => x.BindingInfo?.BindingSource?.IsGreedy != true);
+
+            foreach (var description in notGreedyDescriptions)
+            {
+                var fileParameter = new NonBodyParameter
+                {
+                    Name = description.Name,
+                    In = "formData",
+                    Required = false,
+                    Type = "file"
+                };
+
+                var firstNestedFileParameter = operation
+                    .Parameters
+                    .OfType<NonBodyParameter>()
+                    .FirstOrDefault(x => FormFilePropertyNames.Contains(x.Name));
```

Comment:
Select(Func<T, int, TResult>) + FirstOrDefault() could be used to obtain item and index at once